docs/manuals/source/Appendix/Troubleshooting.rst

```diff
@@ -718,7 +710,7 @@ Solution
 
    ::
 
-      ./bls -j -V DLT-22Apr05 /dev/nst0
+      ./bls -j -V DLT-22Apr15 /dev/nst0
```

Comment:
```suggestion
      bls -j -V DLT-22Apr15 /dev/nst0
```
Normally, the bls command is installed on the system.

Reply:
The changes in the CHANGELOG come from the cherry-pick. Do you prefer to remove those changes?

Reply:
For bls you're right, but we will have to update also master.